client/my-sites/checkout/cart/cart-free-user-plan-upsell.jsx

```diff
@@ -173,9 +171,7 @@ const mapStateToProps = ( state, { cart, addItemToCart } ) => {
 			selectedSiteId &&
 			getPlanPrice( state, selectedSiteId, personalPlan, false ),
 		selectedSite: selectedSite,
-		showPlanUpsell: getCurrentUser( state )
-			? selectedSiteId && currentUserHasFlag( state, NON_PRIMARY_DOMAINS_TO_FREE_USERS )
-			: false,
+		showPlanUpsell: selectedSiteId ?? false,
```

Comment:
I see this warning in the console, which I think comes from this line:

<img width="755" alt="Screen Shot 2022-02-01 at 9 50 47 AM" src="https://user-images.githubusercontent.com/9310939/152002475-0f1424d8-c6db-4a44-a4c4-2f05d7ef7b92.png">